colossalai/zero/comm/zero_comm.py

```diff
@@ -0,0 +1,38 @@
+import torch
+import torch.distributed as dist
+from torch.distributed import ProcessGroup
+from colossalai.context.singleton_meta import SingletonMeta
+from colossalai.utils import get_current_device
+from typing import Optional
+
+import colossal_zero_comm

```

Comment:
try import if failed use the default pytorch API

Reply:
done